src/lib/resources/mail/flags.ts

```diff
@@ -26,3 +24,27 @@ export const {
   },
   shortIDName: "mail address",
 });
+
+export const {
+  flags: mailDeliveryBoxFlags,
+  args: mailDeliveryBoxArgs,
+  withId: withDeliveryBoxId,
+} = makeProjectFlagSet("maildeliverybox", "d", {
+  normalize: async (apiClient, projectId, id): Promise<string> => {
+    if (validateUuid(id)) {
+      return id;
+    }
+
+    const response = await apiClient.mail.listDeliveryBoxes({ projectId });
+    assertStatus(response, 200);
+
+    const deliveryBox = response.data.find(
+      (deliveryBox) => deliveryBox.id === id,
+    );
+    if (!deliveryBox) {
+      throw new Error(`mail delivery box with id "${id}" not found`);
+    }
```

Comment:
This should be unnecessary IMO; effectively, all this does is looking up a delivery box based on its ID, which you can also do using a `getDeliveryBox` operation (which works even better, because that call also can resolve short IDs (at least potentially, because AFAIK that's not implemented yet for mail addresses and delivery boxes).

TLDR; You can probably omit the `normalize` function entirely (not sure if there's a default implementation), because all this function would do is just return the same ID that's passed into it.